Enterprise-CMCS/macpro-mdct-qmr
# MDCT-2765 Pt. II: Add New Definition of Population Validation

### Description

Adding a new validation for definition of population selection to FFY2023 measures

### Related ticket(s)


MDCT-2765

---
### How to test

- Log in as state user
- Navigate to FY2023
- Do not select a `Definition of Population Included in the Measure`, like so:

![Screenshot 2023-07-31 at 2 57 54 PM](https://github.com/Enterprise-CMCS/macpro-mdct-qmr/assets/99458559/8bc8d2c6-8224-4368-b53d-f8cabd113fa0)

Validate the measure and you should see this error message:

![Screenshot 2023-07-31 at 2 58 34 PM](https://github.com/Enterprise-CMCS/macpro-mdct-qmr/assets/99458559/c830b097-a45a-46be-88b5-bc1b61e95c0d)

### Important updates


N/A

---
### Author checklist


---


_convert to a different template: [test → val](?expand=1&template=test-to-val-deployment.md)_ | _[val → prod](?expand=1&template=val-to-prod-deployment.md)_
